tests/suites/test_suite_pkparse.data

```diff
@@ -1008,10 +1004,34 @@ Parse EC Key #4 (PKCS8 DER)
 depends_on:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED
 pk_parse_keyfile_ec:"data_files/ec_prv.pk8.der":"NULL":0
 
+Parse EC Key #4a (PKCS8 DER, no public key)
+depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED
```

Comment:
```MBEDTLS_ECP_C``` is already a dependency of test function ```pk_parse_keyfile_ec``` in ```tests/suites/test_suite_pkparse.function```. Hence not required here and at other places.